Support module path in `--contract-name` (#4420)
foundry-rs/starknet-foundry: docs/src/appendix/sncast/utils/class_hash.md

Required.

The name of the contract. The contract name is the part after the `mod` keyword in your contract file.

> This argument also accepts a module tree path, for example `hello_sncast::contracts::HelloSncast` or `contracts::HelloSncast`., for example `hello_sncast::contracts::HelloSncast`.

Reviewer: ```suggestion
This argument also accepts a module tree path, for example `hello_sncast::contracts::HelloSncast` or `contracts::HelloSncast`.
```

> This argument also accepts an absolute/partial module tree path, for example `hello_sncast::contracts::HelloSncast` or `contracts::HelloSncast`., for example `hello_sncast::contracts::HelloSncast`.

Reviewer 2: ```suggestion
This argument also accepts a module tree path, for example `hello_sncast::contracts::HelloSncast` or `contracts::HelloSncast`, for example `hello_sncast::contracts::HelloSncast`.
```
Also in other places
